src/appComponents/sponsors/SponsorsController.js

```diff
@@ -26,13 +28,15 @@ sponsorsModule.controller('SponsorsController',
         sc.Sponsors = $sessionStorage.sponsors;
 
         if (sc.Sponsors.length === 0) {
+
             ApiJsonFactory.getJson('sponsors')
                 .then(function (response) {
                     sc.Sponsors = response.data.sponsors;
                     $sessionStorage.sponsors = sc.Sponsors;
                 }, function (error) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['error' is defined but never used.](https://www.codacy.com/app/triplez/open-event-webapp/file/2434084761/issues/source?bid=3233857&fileBranchId=3238420#l36)